x11-packages/yad/build.sh

```diff
@@ -0,0 +1,30 @@
+TERMUX_PKG_HOMEPAGE=http://sourceforge.net/projects/yad-dialog/
```

Comment:
Since this is a continuation project the homepage should point at the repo for what's actually being packaged.
```suggestion
TERMUX_PKG_HOMEPAGE=https://github.com/v1cont/yad
```